packages/asset-service/src/service/descriptions.json

```
@@ -1 +1,5 @@
-{}
+{
+  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains CRV and ETH. To acquire Curve CRV-ETH, deposit CRV and/or ETH on https://curve.fi/crveth/deposit. Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss."

```

```suggestion
  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on ShapeShift to earn yield. This liquidity pool contains CRV and ETH. To acquire Curve CRV-ETH, deposit CRV and/or ETH on https://curve.fi/crveth/deposit. Please be aware that as liquidity pools are composed of differently-priced assets, and are subject to impermanent loss."
```
Could also use a semicolon instead of a comma instead of changing the word. (It'd be nice if we could link out to some sort of IL explainer for dummies too, but IDK if these strings support that. Are they Markdown perchance?)

---

oo love the suggestion to add an IL explainer. just made a separate issue to enable markdown hyperlinks https://github.com/shapeshift/web/issues/971

will add the IL explainer once the above issue is merged.

that sentence is worded correctly as is (I think you may have missed the "as"), borrowed straight from Yearn